docs_src/examples/app-service-examples/camera-management/Ch-CameraManagement.md

```diff
@@ -0,0 +1,399 @@
+
+    b. Copy this information into the [add-device-mqtt.yml](https://github.com/edgexfoundry/edgex-compose/blob/{{version}}/compose-builder/add-device-mqtt.yml) file in the `edgex-compose/compose-builder` directory.
+
+    !!! example - "Snippet from add-device-mqtt.yml"
+        ```yaml
+        services:
+        device-mqtt:
+          ...
+          environment:
+            DEVICE_DEVICESDIR: /evam-mqtt-edgex/devices
+            DEVICE_PROFILESDIR: /evam-mqtt-edgex/profiles
+            MQTTBROKERINFO_INCOMINGTOPIC: "incoming/data/#"
+            MQTTBROKERINFO_USETOPICLEVELS: "true"
+          volumes:
+            # example: - /home/github.com/edgexfoundry/edgex-compose/compose-builder/evam-mqtt-edgex:/evam-mqtt-edgex
+            - <add-full-path-of-your-edgex-compose-builder-here-example-above>/evam-mqtt-edgex:/evam-mqtt-edgex
+        ```
+    c. Copy this information into the [add-mqtt-broker.yml](https://github.com/edgexfoundry/edgex-compose/blob/{{version}}/compose-builder/add-mqtt-broker.yml) file in the `edgex-compose/compose-builder` directory.
+
+    !!! example - "Snippet from add-device-mqtt.yml"
+        ```yaml
+        mqtt-broker:
+          volumes:
+            # example: - /home/github.com/edgexfoundry/edgex-compose/compose-builder/evam-mqtt-edgex:/evam-mqtt-edgex
+            - <add-full-path-of-your-edgex-compose-builder-here>/evam-mqtt-edgex/mosquitto.conf:/mosquitto-no-auth.conf:ro
+          ports:
+            - "59001:9001"
+        ```
+
+    !!! note
+        Please note that both the services in this file need the full path to be inserted for their volumes.
+   
+1. Run the following command to start all the Edgex services.
+
+    !!! note
+        The `ds-onvif-camera` parameter can be omitted if no Onvif cameras are present, or the `ds-usb-camera` parameter can be omitted if no usb cameras are present.
+    
+    ```shell
+    make run no-secty ds-mqtt mqtt-broker ds-onvif-camera ds-usb-camera 
+    ```   
+
+### 2. Start [Edge Video Analytics Microservice][evam] running for inference.
+
+!!! note
+    The port for EVAM result streams has been changed from 8554 to 8555 to avoid conflicts with the device-usb-camera service.
+
+```shell
+# Go back to the root of this example app
+cd edgex-examples/application-services/custom/camera-management
+
+# Run this once to download edge-video-analytics into the edge-video-analytics sub-folder, 
+# download models, and patch pipelines
+make install-edge-video-analytics
+```
+
+!!! note
+    If you press `Ctrl-C` it will stop the EVAM services. If you then run `make stop-edge-video-analytics`, it will also remove the containers and free up the port mappings.
+
+### 3. Build and run the example application service
+
+#### 3.1 (Optional) Configure Onvif Camera Credentials.
```

Comment:
Now that the docs are not in the repo, we need a step to clone the example repo so the following steps can be performed there?

Reply:
That's a good point. And that's kind of weird because we usually include that step regardless of where the documentation was held.